modules/standard/Sort.chpl

```diff
@@ -545,13 +626,39 @@ proc isSorted(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator): bool {
 
 @chpldoc.nodoc
 /* Error message for multi-dimension arrays */
+proc isSorted(x: [], comparator:? =new DefaultComparator())
+  where x.domain.rank != 1 {
+    compilerError("isSorted() requires 1-D array");
+}
+
+/*
+   Check if array `Data` is in sorted order
+
+   :arg Data: The array to verify
+   :type Data: [] `eltType`
+   :arg comparator: :ref:`Comparator <comparators>` record that defines how the
+      data is sorted.
+   :returns: ``true`` if array is sorted
+   :rtype: `bool`
+ */
+pragma "last resort"
+@deprecated("'isSorted' with the argument name 'Data' is deprecated, please use the version with the argument name 'x' instead")
+proc isSorted(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator): bool {
+  return isSorted(x=Data, comparator);
+}
+
+
+pragma "last resort"
+@chpldoc.nodoc
+/* Error message for multi-dimension arrays */
+@deprecated("'isSorted' with the argument name 'Data' is deprecated, please use the version with the argument name 'x' instead")
```

Comment:
Does the error overload need a deprecation?

Reply:
You know, that's fair XD